PeerioTechnologies/peerio-mobile
# Feat remove search bar zero state

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/5473/mobile-remove-search-bar-from-the-files-zero-state
#### Testing instructions  
Search bar should no longer show when user has no files or folders.
Search bar should still show if a user does have any files or folders, but is inside an empty folder.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
